monix-execution/shared/src/main/scala_2.13+/monix/execution/compat.scala

```diff
@@ -25,14 +25,13 @@ object compat {
   type BuildFrom[-From, -A, +C] = ScalaBuildFrom[From, A, C]
 
   private[monix] object internal {
-
     type IterableOnce[+X] = scala.collection.IterableOnce[X]
     def toIterator[X](i: IterableOnce[X]): Iterator[X] = i.iterator
     def hasDefiniteSize[X](i: IterableOnce[X]): Boolean = i.knownSize >= 0
 
     def newBuilder[From, A, C](bf: BuildFrom[From, A, C], from: From): mutable.Builder[A, C] = bf.newBuilder(from)
 
-    @inline def toSeq[A](array: Array[AnyRef]): Seq[A] =
+    def toSeq[A](array: Array[AnyRef]): Seq[A] =
```

Comment:
I think `inline` made some difference in some `bufferSliding` benchmarks ( https://github.com/monix/monix/blob/series/4.x/benchmarks/shared/src/main/scala/monix/benchmarks/ChunkedEvalFilterMapSumBenchmark.scala#L123 ) but I'm not 100% sure

Reply:
Maybe. But it's risky, and I'd rather get rid of these. We can redo the benchmarks, and then add some explicit comments for that inlining, to ensure that we did our homework. 

What I want to avoid is overuse of the `inline` annotation, which in the beginning I believed that it was a good idea, and added such annotations based on gut feeling.

Reply:
Makes sense, I'll just rerun the benchmark before releasing